modules/gitops-inbuilt-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-inbuilt-permissions-for-cluster-config_{context}"]
+= In-built permissions for cluster configuration
+
+The default Argo CD instance has some default permissions to manage specific cluster-scoped resources which include platform operators, optional OLM operators and user management. 
+
+[NOTE]
+====
+Argo CD will not have cluster-admin permissions.
```

Comment:
Future tense should be avoided. Suggested:

"Argo CD does not have cluster-admin permissions."